aspnet/fundamentals/localization/sample/Controllers/BookController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Localization;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Localization;
+
+namespace Localization.StarterWeb.Controllers
+{
+    public class BookController : Controller
+    {
+        private readonly IHtmlLocalizer<BookController> _localizer;
+
+        public BookController(IHtmlLocalizer<BookController> localizer)
+        {
+            _localizer = localizer;
+        }
+
+        public IActionResult Hello(string name)
+        {
+            ViewData["Message"] = _localizer["<b>Hello</b><i> {0}</i>", name];
+
+            return View();
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        public IActionResult About()
+        {
+            IRequestCultureFeature requestCultureFeature = HttpContext.Features.Get<IRequestCultureFeature>();
+            RequestCulture requestCulture = requestCultureFeature.RequestCulture;
+            ViewData["requestCultureFeature"] = requestCultureFeature;
+            ViewData["requestCulture"] = requestCulture;
+            ViewData["Message"] = _localizer["Your application description page."];
+
+            return View();
+        }
+
+        //title="@Localizer["Log in using your {0} account", @provider.DisplayName]"> 
```

Comment:
Added explanation 

        // Pass the requestCultureFeature and requestCulture objects to the view
        // so we can get Culture/UICulture information.